src/redux/actions/index.js

```diff
@@ -14,6 +17,20 @@ export function setCategories(categories) {
   };
 }
 
+export function setLearntCategories(learntcategories) {
+  return {
+    type: SET_LEARNT_CATEGORIES,
+    payload: learntcategories,
+  };
+}
+
+export function setCurrentLearntCategory(learntcategoryId) {
```

Comment:
```suggestion
export function setCurrentLearntCategory(learntCategoryId) {
```
Use consistent casing